dist/svg.js

```diff
@@ -2414,15 +2415,28 @@ SVG.Matrix = SVG.invent({
     }
     // Skew
   , skew: function(x, y, cx, cy) {
-      return this.around(cx, cy, this.native().skewX(x || 0).skewY(y || 0))
+      // support uniformal skew
+      if (arguments.length == 1) {
+        y = x
+      } else if (arguments.length == 3) {
+        cy = cx
+        cx = y
+        y = x
+      }
+
+      // convert degrees to radians
+      x = SVG.utils.radians(x)
+      y = SVG.utils.radians(y)
+
+      return this.around(cx, cy, new SVG.Matrix(1, Math.tan(y), Math.tan(x), 1, 0, 0))
```

Comment:
Something I should really have mentioned is the problem that this is fixing. The following code should illustrate:

```javascript
var rect = draw.rect(100,100)

// Before this commit
rect.skew(14, 20)
rect.transform().skewX // return 14 as expected
rect.transform().skewY // return 18.453269962755858 instead of 20

// With this commit
rect.skew(14, 20)
rect.transform().skewX // return 14 as expected
rect.transform().skewY // return 20 as expected
```